configuration.rst

```diff
@@ -131,6 +133,20 @@ db-extra-search-path
 
   Multiple schemas can be added in a comma-separated string, e.g. ``public, extensions``.
 
+.. _db-channel:
+
+db-channel
+----------
+
+  The name of the notification channel that PostgREST uses to reload the schema cache. For more information, see :ref:`schema_reloading`.
```

Comment:
Note that the db-channel is also used for reloading the config. Check https://github.com/PostgREST/postgrest-docs/issues/409.